patches/xliff-tasks/0002-remove-netcoreapp2.0-targets.patch

```diff
@@ -0,0 +1,48 @@
+From d2c2c5a7e5472973d2dc843026087ec69c45b53b Mon Sep 17 00:00:00 2001
+From: Tom Deseyn <tom.deseyn@gmail.com>
+Date: Tue, 17 Nov 2020 08:34:21 +0100
+Subject: [PATCH] xliff-tasks: remove netcoreapp2.0 targets
+
+---
+ src/XliffTasks/XliffTasks.csproj        | 4 ++--
+ src/XliffTasks/build/XliffTasks.targets | 2 +-
+ 2 files changed, 3 insertions(+), 3 deletions(-)
+
+diff --git a/src/XliffTasks/XliffTasks.csproj b/src/XliffTasks/XliffTasks.csproj
+index d967ac3..38bd62b 100644
+--- a/src/XliffTasks/XliffTasks.csproj
++++ b/src/XliffTasks/XliffTasks.csproj
+@@ -4,7 +4,7 @@
+ 
+   <PropertyGroup>
+     <TargetFrameworks>netcoreapp2.0;net46</TargetFrameworks>
+-    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.0</TargetFrameworks>
++    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">net5.0</TargetFrameworks>
+     <BuildOutputTargetFolder>tools</BuildOutputTargetFolder>
```

Comment:
Given this patch, I'm surprised to still see this in the prebuilts:
```
<Usage Id="Microsoft.NETCore.DotNetHostResolver" Version="2.0.0" File="src/xliff-tasks.a52f3d7fb58470749ee4035fbbcb7e63c78b0459/artifacts/obj/XliffTasks/project.assets.json"/>
```

Reply:
Hmm. Looks like it's added intentionally by the project itself as far as NuGet knows:

```
  "projectFileDependencyGroups": {
    "net5.0": [
      "Microsoft.Build.Framework >= 15.1.1012",
      "Microsoft.Build.Utilities.Core >= 15.1.1012",
      "Microsoft.NETCore.App >= 2.0.0",
      "Microsoft.SourceLink.AzureRepos.Git >= 1.0.0",
      "Microsoft.SourceLink.GitHub >= 1.0.0"
    ]
  },
```

So this might be xliff-tasks doing something weird, or the SDK adding it as an implicit reference for a bad reason.

Looks like the project file in this case:

```
  <ItemGroup Condition="'$(TargetFramework)' == 'net5.0'">
    <PackageReference Include="Microsoft.Build.Framework" Version="15.1.1012" PrivateAssets="All" />
    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.1.1012" PrivateAssets="All" />
    <PackageReference Include="Microsoft.NETCore.App" Version="2.0.0" PrivateAssets="All" />
  </ItemGroup>
```

The line can most likely just be deleted.

Reply:
Wow, CI didn't like that at all. 😄 I guess it's relying on that package reference because the project says `<DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>`? The refs should come from `packs/Microsoft.NETCore.App.Ref/5.0.0-.../...` in the SDK now (as of 3.0+). I'm not completely sure how to manually force those references to get added. I can't think of a reason we couldn't just remove `DisableImplicitFrameworkReferences`...